packages/react-components/review/beta/react-components.api.md

```diff
@@ -209,18 +209,23 @@ export interface CameraSitePermissionsProps extends CommonSitePermissionsProps {
 // @beta
 export type CameraSitePermissionsStrings = SitePermissionsStrings;
 
+// Warning: (ae-internal-missing-underscore) The name "CaptionInfo" should be prefixed with an underscore because the declaration is marked as @internal
+//
 // @internal
-export const _Caption: (props: _CaptionProps) => JSX.Element;
+export type CaptionInfo = {
```

Comment:
Caption**s**Info
If this is just an internal class, add underscore to get rid of warnings